### DISCUSSION TITLE
请问需要Qos怎么办

### DISCUSSION THREAD

**Discussion by DemonJimmy** [istoreos/istoreos#1403] [Q&A]
如题，最近下了docker版的网心云，晚高峰吃的流量太狠导致完全没办法正常上网，不知道为什么也没办法直接进入网心云后台修改速度什么的也估计没有。商店内的IP限速似乎也无效或者我也不知道docker从属的是哪个应该是169那个？尝试了也无效，所以请问如何安装类似路由器一样的Qos服务或者如何对他进行限速，最好是有Qos的帮助。
OP好像是自带Qos的，如果可能，也希望官方能在未来加入Qos
X86实体机版本

**Comment by jjm2473** [comment]
istore有个ip限速插件，但不知道对docker有没有效果

**Comment by w9315273** [comment]
想要精细控制流量就安装tc 用法自己搜

**Comment by gbeta** [comment]
解决了吗 我也遇到这个问题了，概述里能看到上行，但是就是找不到docker的网心，从而没有办法限制它
